wp-content/themes/access/views/single-page.twig

```diff
@@ -43,7 +46,7 @@
       </aside>
       {% endif %}
 
-      <div class="o-content">
+      <div class="o-content" itemtype="https://schema.org/about">
```

Comment:
`itemtype`s should be `itemprop` within the scope of the `WebPage` schema and you can drop the url.

```
itemprop="about">
```